assets/js/base/context/cart-checkout/payment-methods/payment-method-data-context.js

```diff
@@ -445,6 +449,8 @@ export const PaymentMethodDataProvider = ( { children } ) => {
 		errorMessage: paymentData.errorMessage,
 		activePaymentMethod,
 		setActivePaymentMethod,
+		activeSavedToken,
+		setActiveSavedToken,
```

Comment:
You'll need to do a couple things here:

- update the typedefs in for the `PaymentMethodDataContext` in `type-defs/contexts`
- make sure `DEFAULT_PAYMENT_METHOD_DATA` in `base/context/cart-checkout/payment-methods/constants.js` is updated with the new properties (and an appropriate default value).

Reply:
Done.